rwth_upper_body_skeleton_random_walk/msg/SkeletonDetector.msg

```diff
@@ -0,0 +1,3 @@
+Header header
+float64[] pos_3d 	# upper body skeleton joint positions
+
```

Comment:
Instead of your own message, maybe just using a `geometry_msgs/PoseArray` would be easier and more standardised?